### PR TITLE
Removing bad refs to 'Account settings UI' - DOC-7091

### PR DESCRIPTION
**Pull request by zuluecho9**
See DOC-7091 for more info. For peer edit: it may be hard to parse this because it's related to a lot of complex v1/v2 user model stuff and how that interacts with some product features. FWIW, I'm feeling good about it.  

Summary of work: 
* Removing bad/inaccurate references to the 'Account settings' tab in the account dropdown UI because v2 users can't see that. Some instances remain for places we need that for v1 users. 
* Assorted edits related to this, including making some places more vague/general to futureproof us a bit. 
* Separating 'account dropdown' from 'account switcher' in the glossary doc.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings.mdx [hunk 1]
```diff
@@ -7,68 +7,39 @@ tags:
...
 Keep reading for tips on how to find different kinds of account settings in the UI. 
 
-## Pricing and billing settings [#pricing]
+## Pricing, billing, and usage UI [#pricing]
 
-To view and edit some billing-related settings, you must be a [full user](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type). Basic users have some restrictions.
+Note that [user permissions](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/) may impact your access to some account settings. 
 
-How you access billing-related UI depends on which [pricing plan](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#pricing-plans) your New Relic account is on: 
+To find billing-related UI, go to [one.newrelic.com](https://one.newrelic.com) and, in the upper right corner, click the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown) to access various account settings. Here are the billing-related UI pages available there:
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings.mdx:18]: "Here are the billing-related UI pages available there:" I prefer something more like:

"Here's what you can do there:"

File: src/content/docs/accounts/accounts/saml-single-sign/maintain-sso-settings.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ watermark: |-
 ---
 
 <Callout variant="important">
-  These docs are for setting up SSO for users on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#how-pricing-users-relate). For SSO for users on New Relic One user model, see [Authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso).
+  These docs are for setting up SAML SSO for users on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#how-pricing-users-relate). If you're on the New Relic One user model, see [Authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso).
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts/saml-single-sign/maintain-sso-settings.mdx:20]: "These docs are for..." --> "These docs describe..."

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure.mdx [hunk 3]
```diff
@@ -94,7 +94,7 @@ How to set up SAML SSO will depend on which [user model](/docs/accounts/original
 **Requirements:**
 
 * Pro or Enterprise [edition](https://newrelic.com/pricing).
-* Your must be on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (if you're on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models), you need assistance from New Relic).
+* Your must be on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (if you're on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models), you need our assistance).
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure.mdx:97]: "Your" --> "You"

Comment (paperclypse) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure.mdx:97]: It seems weird to say "you need our assistance" without telling someone how to get that assistance.

**Review by paperclypse — DISMISSED** [review]
Looks great! I left a couple of minor comments, but these changes all look good.

**Comment by zuluecho9** [comment]
thank you much @paperclypse !
